sdk/documentintelligence/Azure.AI.DocumentIntelligence/CHANGELOG.md

```diff
@@ -3,10 +3,46 @@
 ## 1.0.0-beta.4 (Unreleased)
 
 ### Features Added
-
+- Added methods `GetAnalyzeBatchResult`, `GetAnalyzeBatchResults`, `DeleteAnalyzeBatchResult`, and `DeleteAnalyzeResult` to `DocumentIntelligenceClient`.
+- Added class `AnalyzeBatchOperationRename` to be used as the output of the `GetAnalyzeBatchResult` and `GetAnalyzeBatchResults` APIs.
+- Added property `ModifiedOn` to `DocumentModelDetails` and to `DocumentClassifierDetails`.
+- Added member `Skipped` to `DocumentIntelligenceOperationStatus` (former `OperationStatus`).
 - Exposed `JsonModelWriteCore` for model serialization procedure.
 
 ### Breaking Changes
+- `DocumentIntelligenceClient` and `DocumentIntelligenceAdministrationClient` now target service API version `2024-11-30`. Support for `2024-07-31-preview` has been removed.
```

Comment:
These are not breaking changes if they occur between betas.   By convention, we do not document changes like this between betas, as they're expected to occur and this makes it much harder to see important breaking cases for GA versions.

Reply:
Removed this change.